src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto

```diff
@@ -541,6 +541,9 @@ message BidExt {
         // Right marketplace. Omitted for open auction.
         optional string pmptier = 10;
 
+        //OD15838 Temporary field to allow for transition period
+        optional int32 pmptier_int = 15;
```

Comment:
Personally I'd say better to put this below `imptrackers` to keep field ordering consistent (as long as comment points to existing `pmptier`)